### PR TITLE
add new logo and modify a bit the index page introduction 

### PR DESCRIPTION
**Pull request by philippelaumay**

### CODE REVIEW

File: md/index.md [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-![Bonita BCD](images/bcd_overview.png)
+![Bonita CD](images/BCD.png)
 
-This module provides a solution to use _Continuous Delivery_ practice for your Bonita Living Application development.
+This add-on provides a solution to use _Continuous Delivery_ practice for your Bonita Living Application development and deployment.
 It is composed of two main parts:
-* First a framework for continuous integration of Living Application.
-* Second a solution for Cloud environment provisioning.  
+* First a solution for Bonita stack provisioning on Cloud or on-premise.  
+* Second a framework for continuous integration of Living Application. _this CI framework will be delivered in future versions of the add-on_
```
Comment (tnguyen1) [inline md/index.md]: Capital "T" for "this CI framework ..."

File: md/index.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
 
-This documentation focus on the second part: the solution and tooling for provisioning of a Bonita stack.
-The tool is composed of a set of scripts to deploy Bonita platform automatically (database and Bonita Tomcat application server).
+![Bonita CD logo](images/BCD_logo.png) This documentation focus on the first part: the solution and tooling for provisioning of a Bonita stack.
```
Comment (tnguyen1) [inline md/index.md]: Not sure of the position of the logo - please check http://doc.rd.lan:10017/bcd/1.0/

**Review by tnguyen1 — CHANGES_REQUESTED** [review]

File: md/index.md (hunk 2 above)
Comment (philippelaumay) [reply md/index.md]: right, I'll change that...

File: md/index.md [hunk 3]
```diff
@@ -1,24 +1,24 @@
...
+The tool is composed of a set of scripts to deploy a full Bonita stack automatically (database and Bonita Tomcat application server).
 
-This toolkit is compatible with Amazon EC2 cloud platform, on premises servers and local virtual machines via [Vagrant](https://www.vagrantup.com/).
+This toolkit is compatible with Amazon EC2 cloud platform (AWS), on premises servers and local virtual machines via [Vagrant](https://www.vagrantup.com/).
 
 These scripts are powered by [Ansible](http://www.ansible.com). Short note from Wikipedia:
 
 >Ansible is a free software platform for configuring and managing computers. It combines multi-node software deployment, ad hoc task execution, and configuration management. It manages nodes over SSH or PowerShell and requires Python (2.4 or later) to be installed on them. Modules work over JSON and standard output and can be written in any programming language. The system uses YAML to express reusable descriptions of systems.
```
Comment (tnguyen1) [inline md/index.md]: This paragraph renders with a bigger font apparently - maybe we could put it within an "info" block like this:
```
::: info
Ansible is a free software platform for configuring and managing computers. It combines multi-node software deployment, ad hoc task execution, and configuration management. It manages nodes over SSH or PowerShell and requires Python (2.4 or later) to be installed on them. Modules work over JSON and standard output and can be written in any programming language. The system uses YAML to express reusable descriptions of systems.
:::

**Review by tnguyen1 — APPROVED** [review]

**Pull request merged**